inc/class-emitter.php

```diff
@@ -313,6 +314,7 @@ public static function get_rest_api_surrogate_keys() {
 		 */
 		$keys = self::get_instance()->rest_api_surrogate_keys;
 		$keys = array_unique( $keys );
```

Comment:
Is this `array_unique` unnecessary since we're doing it again on 319?
🔢

Reply:
Probably? Can test.

Reply:
I've been testing removing this this morning, and haven't come across any issues. Pushed commit [dc0c290](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/228/commits/dc0c290c41ad22da9f18f3378c37b52432124fef) removing the duplicate.